Update optimizer-hints.md (#3966)
pingcap/docs: optimizer-hints.md

select /*+ READ_FROM_STORAGE(TIFLASH[t1], TIKV[t2]) */ t1.a from t t1, t t2 where t1.a = t2.a;
```

> **Note:**
>

> > If the table to be read by the optimizer hint is not from the same schema, you need to declare the schema name. For example:

Reviewer: ```suggestion
> To tell the optimizer that the table to be read is from another schema, you need to explicitly specify the schema name. For example:
```

> > To tell the optimizer that the table to be read is from another schema, you need to explicitly specify the schema name. For example:

Reviewer: ```suggestion
> If you want the optimizer to use a table from another schema, you need to explicitly specify the schema name. For example:
```

> 
> {{< copyable "sql" >}}
> 
> ```sql
> `SELECT /*+ READ_FROM_STORAGE(TIFLASH[test1.t1,test2.t2]) */ t1.a FROM test1.t t1, test2.t t2 WHERE t1.a = t2.a;`
> ```

Reviewer: ```suggestion
>
> {{< copyable "sql" >}}
>
> ```sql
> SELECT /*+ READ_FROM_STORAGE(TIFLASH[test1.t1,test2.t2]) */ t1.a FROM test1.t t1, test2.t t2 WHERE t1.a = t2.a;
> ```
```
